package.json

```diff
@@ -21,10 +27,9 @@
   },
   "scripts": {
     "lint": "eslint --fix .",
-    "pretest": "npm run lint",
     "test": "ava",
     "test:watch": "ava --watch",
-    "prebuild": "npm run test",
+    "prebuild": "npm run lint && npm run test",
```

Comment:
It's a way to ensure the build (and thus `publish`) will fail if tests are not passing. I could go with `prebulish: "npm run lint && npm run test && npm run build"` but felt like it makes sense to prevent invalid builds.